metricbeat/module/envoyproxy/_meta/fields.yml

```diff
@@ -3,6 +3,7 @@
   description: >
     envoyproxy module
   release: ga
+  settings: ["ssl", "http"]
```

Comment:
great to see this reusable component. @mtojek @ycombinator Would be great to have something like this also for the build process of integrations to pull in shared doc snippets.